.travis.yml

```diff
@@ -7,11 +7,12 @@ php:
 before_script:
   - composer self-update
   - composer update
+  - wget https://phpbench.github.io/phpbench/phpbench.phar https://phpbench.github.io/phpbench/phpbench.phar.pubkey
```

Comment:
Please use a `composer require` step in a separate build pipeline entry

Reply:
copied `doctrine/dbal` here, FYI.

Reply:
you prefer to `require` in the travis step, or in `composer.json`?